lms/static/js/demographics_collection/DemographicsCollectionModal.jsx

```diff
@@ -0,0 +1,460 @@
+        // decline was just selected
+      } else if (decline) {
+        return { selected: { ...selected, [FIELD_NAMES.ETHNICITY]: [decline] } }
+        // anything else was selected
+      } else {
+        return { selected: { ...selected, [FIELD_NAMES.ETHNICITY]: values } }
+      }
+    });
+  }
+
+  handleInputChange(e) {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState(prevState => ({
+      selected: {
+        ...prevState.selected,
+        [name]: value,
+      }
+    }));
+  }
+
+  // We need to transform the ethnicity array before we POST or after GET the data to match
+  // from [{ethnicity: 'example}] => to ['example']
+  // the format the UI requires the data to be in.
+  reduceEthnicityArray(ethnicityArray) {
+    return ethnicityArray.map((o) => o.ethnicity);
+  }
+
+  render() {
+    if (this.state.loading) {
+      return <div className="demographics-collection-modal d-flex justify-content-center align-items-start" />
+    }
+    return (
+      <div className="demographics-collection-modal d-flex justify-content-center align-items-start">
+        <Wizard
+          onWizardComplete={this.props.closeModal}
+          dismissBanner={this.props.dismissBanner}
+          wizardContext={{ ...this.state.selected, options: this.state.options }}
+          error={this.state.error}
+        >
+          <Wizard.Header>
+            {({ currentPage, totalPages }) => (
+              <div>
+                <p className="font-weight-light">
+                  {StringUtils.interpolate(
+                      gettext('Section {currentPage} of {totalPages}'),
+                        {
+                          currentPage: currentPage,
+                          totalPages: totalPages
+                        }
+                      )
+                  }
+                </p>
+                <h2 className="mb-1 mt-4 font-weight-bold text-secondary">
+                  {gettext('Help make edX better for everyone!')}
+                </h2>
+                <p>
+                  {gettext('Welcome to edX! Before you get started, please take a few minutes to fill-in the additional information below to help us understand a bit more about your background. You can always edit this information later in Account Settings.')}
+                </p>
+                <br />
+                <span className="fa fa-info-circle" />
+                {/* Need to strip out extra '"' characters in the marketingSiteBaseUrl prop or it tries to setup the href as a relative URL */}
+                <a className="pl-3" href={`${this.props.marketingSiteBaseUrl}/demographics`.replace(/"/g, "")}>
+                  {gettext('Why does edX collect this information?')}
+                </a>
+                <br />
+                <p>
+                  {StringUtils.interpolate(
+                      gettext('Part {currentPage} of {totalPages}'),
```

Comment:
Not sure how this got changed back, bit I think the mock shows `Section {currentPage} of {totalPages}'

Reply:
I've deleted this twice now 😓

Reply:
🛠️ 'd